### ISSUE TITLE
Error while running azdev setup -c

### ISSUE THREAD

**Issue by arde0708** [Azure/azure-cli#14715]
Update : I have been able to resolve this issue now but running into another issue which is described in the comment section

------------------------------------------------------------------------------------------------------------------------
I am trying to setup the CLI environment for development. This is the doc I am referring : https://github.com/Azure/azure-cli/blob/master/doc/configuring_your_machine.md

I have been able to execute all the steps successfully so far except Step 6ii)
Activate your environment and execute the following commands
pip install azdev
**azdev setup -c**
Python Interpreter : 3.7.4 64 Bit

I am running into the below error when I run the 'azdev setup -c' command from VSCode. 
`PS F:\aug7\azure-cli> azdev setup -c Traceback (most recent call last):   File "d:\users\arde\appdata\local\programs\python\python37\lib\runpy.py", line 193, in _run_module_as_main     "__main__", mod_spec)   File "d:\users\arde\appdata\local\programs\python\python37\lib\runpy.py", line 85, in _run_code     exec(code, run_globals)   File "D:\Users\arde\AppData\Local\Programs\Python\Python37\Scripts\azdev.exe\__main__.py", line 9, in <module>   File "d:\users\arde\appdata\local\programs\python\python37\lib\site-packages\azdev\__main__.py", line 40, in main     config_dir=get_azdev_config_dir())   File "d:\users\arde\appdata\local\programs\python\python37\lib\site-packages\azdev\utilities\config.py", line 24, in get_azdev_config_dir     _, env_name = os.path.splitdrive(get_env_path())   File "d:\users\arde\appdata\local\programs\python\python37\lib\ntpath.py", line 141, in splitdrive     p = os.fspath(p) TypeError: expected str, bytes or os.PathLike object, not NoneType`

VSCode Version: 1.47.3 (user setup)
Commit: 91899dcef7b8110878ea59626991a18c8a6a1b3e
Node.js: 12.8.1

**Comment by yonzhan** [comment]
add to S174

**Comment by arde0708** [comment]
I have been able to resolve this issue ( by running the command after creating and enabling the virtual environment ). However, I am running into a new issue now. I have cloned this repo, and have been noticing too many errors even without my changes. Can someone help with the additional steps I need to perform before being dev ready ?

There are the steps I have followed so far for VS Code Section here : 
https://github.com/Azure/azure-cli/blob/dev/doc/configuring_your_machine.md

Here are the errors :

![image](https://user-images.githubusercontent.com/63367896/89700644-b17b1c80-d8e4-11ea-865c-fea561cf22f6.png)
